runtime/centrifuge/src/lib.rs

```diff
@@ -2099,7 +2097,7 @@ construct_runtime!(
 		LiquidityRewardsBase: pallet_rewards::<Instance2>::{Pallet, Storage, Event<T>, Config<T>} = 104,
 		LiquidityRewards: pallet_liquidity_rewards::{Pallet, Call, Storage, Event<T>} = 105,
 		GapRewardMechanism: pallet_rewards::mechanism::gap = 106,
-		LiquidityPoolsGateway: pallet_liquidity_pools_gateway::{Pallet, Call, Storage, Event<T>, Origin } = 107,
+		LiquidityPoolsGateway: pallet_liquidity_pools_gateway::{Pallet, Call, Storage, Event<T> } = 107,
```

Comment:
I think this could imply a migration because we're modifying the origin enum stored by the `order_book`, through the `Feeder` wrapper 🤔. How can we check if this happens? `try-runtime` should tell this automatically?

Reply:
Possible, yes. Thanks for bringing it up! I added it to the post audit migration [todo list](https://centrifuge.hackmd.io/_bJWZVKySHOCDkhrpbqbAQ?both#Post-audit).